src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistor.java

```diff
@@ -64,7 +64,7 @@
 import org.jooq.exception.DataAccessException;
 import org.jooq.impl.DSL;
 
-class SQLitePersistor extends PersistorBase {
+public class SQLitePersistor extends PersistorBase {
```

Comment:
We might not want to make it public. The idea is all users should use the public interface rather than actual DB implementor. Would you mind your usecase for this change ?

Reply:
Using @VisibleForTesing. I am creating a sqlitePersistor for my test purposes.

Reply:
I was using the sqlitePersistor for my test purposes. Moved the test, I no more require it there. Removed this